.github/workflows/update_metdata.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup environment
         run: |
           pip install --upgrade pip
-          pip install datasets pandas==2.0.3
+          pip install datasets pandas
```

Comment:
If we go look in the [job](https://github.com/huggingface/transformers/actions/runs/13967690998/job/39101790937), `torch==2.1.2` is being installed, due to the pinned pandas version.

`utils/update_metadata.py` runs with `pandas==2.2.2`, so the pin should not be needed anymore :)